src/components/ToDoList.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import ToDo from './ToDo';
+
+function ToDoList({
+  toDoList,
+  onClickDelete,
+}) {
+  if (toDoList.length === 0) { return (<p>할 일이 없어요!</p>); }
```

Comment:
```suggestion
  if (toDoList.length === 0) { 
    return <p>할 일이 없어요!</p> 
  }
```

취향 차이인거같은데! 저는 if 문은 항상 한줄 띄는걸 좋아합니다! :)

Reply:
참고했던 guard clause 예시들이 붙어있길래 따라해 봤었는데요.. 딱히 취향이 될 거 같지는 않아서 한줄 띄는 일반적 패턴으로 저도 돌아가렵니다.